azure-ts-functions-raw/README.md

```diff
@@ -1,6 +1,6 @@
 [![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
 
-# Azure Functions
+# Azure Functions in .NET and Java
```

Comment:
There are like six languages in there. List them all or replace with "all supported languages"?